llama-index-integrations/vector_stores/llama-index-vector-stores-upstash/llama_index/vector_stores/upstash/base.py

```diff
@@ -121,7 +123,11 @@ def client(self) -> Any:
         return self._index
 
     def __init__(
-        self, url: str, token: str, batch_size: int = DEFAULT_BATCH_SIZE
+        self,
+        url: str,
+        token: str,
+        batch_size: int = DEFAULT_BATCH_SIZE,
+        namespace: Optional[str] = "",

```

Comment:
nit: can we add a comment here that `""` is the default namespace used by Upstash if none is provided

Reply:
also, we don't need the `Optional` here either right? We actually don't want this to be `None` or `str` type. This is strictly a `str` type with a default `""` if no string is provided by the user.

Reply:
Yeah that makes sense, updated it.